samples/dect/dect_phy/dect_shell/src/dect/perf/dect_phy_perf.c

```diff
@@ -1062,6 +1068,7 @@ static int dect_phy_perf_tx_request_results(void)
 	memcpy(&phy_header.type_2, &header, sizeof(phy_header.type_2));
 
 	perf_pdu.header.message_type = DECT_MAC_MESSAGE_TYPE_PERF_RESULTS_REQ;
+	perf_pdu.message.results_req.foo = 0;
```

Comment:
I started wondering about `foo` and then found also `foo_fighters` in `dect_phy_perf_pdu_harq_feedback_t`. The naming could be more descriptive, I guess. 😄